test/unit/config/index.js

```diff
@@ -338,6 +271,166 @@ describe('Config', () => {
             });
         });
     });
+
+    describe('rendering', () => {
```

Comment:
Suggest reorganising this again :)
How about renaming `describe('options validation', () => {` to `describe('options options', () => {` and check all of the options one by one? Probably, arranging a single describe for each?